process_noise_covariance: [0.01, 0, 0, 0, 0, 0, 0, 0, 0,
                           0, 0.01, 0, 0, 0, 0, 0, 0, 0,
                           0, 0, 0.01, 0, 0, 0, 0, 0, 0,
                           0, 0, 0, 0.1, 0, 0, 0, 0, 0,
                           0, 0, 0, 0, 0.1, 0, 0, 0, 0,
                           0, 0, 0, 0, 0, 0.1, 0, 0, 0,
                           0, 0, 0, 0, 0, 0, 0.1, 0, 0,
                           0, 0, 0, 0, 0, 0, 0, 0.1, 0,
                           0, 0, 0, 0, 0, 0, 0, 0, 0.1]

gps_covariance: [1.5, 0, 0,
                 0, 1.5, 0,
                 0, 0, 1.5]

accel_covariance: [0.1, 0, 0,
                   0, 0.1, 0,
                   0, 0, 0.1]

mag_covariance: [0.1, 0, 0,
                 0, 0.1, 0,
                 0, 0, 0.1]

vel_covariance: [0.1, 0, 0,
                0, 0.1, 0,
                0, 0, 0.1]